src/main/java/net/neoforged/moddevgradle/internal/ModDevPlugin.java

```diff
@@ -575,7 +585,7 @@ private List<Configuration> configureArtifactManifestConfigurations(Project proj
             spec.setDescription("Dependencies needed for running NeoFormRuntime for the selected NeoForge/NeoForm version (Classpath)");
             spec.setCanBeConsumed(false);
             spec.setCanBeResolved(true);
-            spec.getDependencies().addLater(neoForgeDependency); // Universal Jar
+            spec.getDependencies().addLater(extension.getNeoForgeArtifact().map(a -> a + ":universal").map(dependencyFactory::create)); // Universal Jar
```

Comment:
The resolution is done with the capability, the `universal` classifier should be removed.